community/keychain/build

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh -e
+
+
+
+mkdir -p "$1/usr/bin"
+
+install -m755 "mksh" "$1/usr/bin/mksh"
```

Comment:
```
 install -Dm755 "mksh" "$1/usr/bin/mksh"
```

Add `-D` and also fix this as it is from `mksh`.

Reply:
lol. must have needed moar coffee when doing this one eh. fixed